spring-cloud-dataflow-docs/src/main/asciidoc/configuration-local.adoc

```diff
@@ -196,6 +196,173 @@ spring:
     driver-class-name:org.postgresql.Driver
 ----
 
+[start=4]
+. Or build a custom server with your own build files. Here are examples of bundling MySQL driver using gradle and maven, which
+will result same jar as `2.1.2.RELEASE` with addition of a `mysql-connector-java-8.0.16.jar`:
+
+[source, groovy]
+.gradle
```

Comment:
Should show an empty project with the gradle and maven files in it? Maybe easy that way for folks to copy the file to repeat it.

If you think it is a good idea, we could host that empty project in [samples](https://github.com/spring-cloud/spring-cloud-dataflow-samples), and cross-link it here.

Reply:
Created https://github.com/spring-cloud/spring-cloud-dataflow-samples/issues/107 to track this.